README.md

```diff
@@ -1,14 +1,17 @@
-# SIGIO
+![Status](https://github.com/NOAA-EMC/NCEPLIBS-sp/workflows/Build%20and%20Test/badge.svg)
 
-Does something with Spectral Coefficients!
+# SIGIO
 
+Does something with Spectral Coefficients. This is part of the
```

Comment:
This looks leftover from a copy-paste from the SP library.

I found this in the code that seems like a better description:

```
This library provides an Application Program Interface
 for performing I/O on the sigma restart file of the global spectral model.
 Functions include opening, reading, writing, and closing as well as
  allocating and deallocating data buffers used in the transfers.
  The I/O performed here is sequential.
  The transfers are limited to header records or data records.
```

Reply:
Excellent, thanks. I will use this...